### ISSUE TITLE
I love you!

### ISSUE THREAD

**Issue by nacaseven** [1kc/razer-macos#113]
I just found this app and... thank you so much! Since Catalina my Razer Setup was a colour-cycling unusable mess. 
I REALLY want to donate to you guys, how can I do this?

One thing I'd love to see in a future update --> brightness control!

**Comment by HenkPoley** [comment]
Yes, I tried testing this with my Razer Viper Ultimate, and the light doesn't turn on. 

Which is incidentally the way I like it. So I don't mind. But I was looking for an app to turn the light off, in case it ever turned on again 😅

Razer macOS does change the DPI successfully, so it can talk with the hardware, on Apple Silicon 👍

**Comment by Freivincampbell** [comment]
> Yes, I also tried testing this app with my Razer Viper Ultimate, and the light doesn't turn on.
> 
> Which is incidentally the way I like it. So I don't mind. But I was looking for an app to turn the light off, in case it ever turned on again 😅. I'm pretty sure my mouse now has a 'color', just with the lowest brightness.
> 
> Razer macOS does change the DPI successfully, so it can talk with the hardware, from Rosetta 2 on Apple Silicon 👍. Razer Synapse 2.0 can't do that.

adasdasd

**Comment by Freivincampbell** [comment]
Yep @HenkPoley  I got the same problem the DPI set up does not work for me, it opens only a black windows

**Comment by 1kc** [comment]
Cheers! Yeah brightness control would be good once I find some spare time to do it, right now I am mostly integrating/merging features from other contributors.

@Freivincampbell that issues should be fixed btw

**Comment by 1kc** [comment]
@nacaseven yay brightness control has been added for keyboards!

**Comment by Freivincampbell** [comment]
@1kc Thank you 😊 so much

**Comment by nacaseven** [comment]
Awesome work! Thank you! :)

**Comment by joeyiscool12** [comment]
@1kc  hey, sorry to bother you but my razer viper ultimate doesn't respond when I change the colors in the software, I have restarted my computer, switched memory profiles, and tried uninstalling and reinstalling. Please try to fix this, thanks!
